src/caret_analyze/architecture/architecture.py

```diff
@@ -77,9 +77,11 @@ def get_node(self, node_name: str) -> NodeStructValue:
     def get_executor(self, executor_name: str) -> ExecutorStructValue:
         return Util.find_one(lambda x: x.executor_name == executor_name, self.executors)
 
-    def get_callback_group(self, callback_group_name: str) -> CallbackGroupStructValue:
-        return Util.find_one(lambda x: x.callback_group_name == callback_group_name,
-                             self.callback_groups)
+    def get_callback_group(self) -> tuple[CallbackGroupStructValue, ...]:
+        cbg: list[CallbackGroupStructValue] = []
+        for node in self.nodes:
+            cbg += node.callback_groups
+        return tuple(set(cbg))
```

Comment:
get_callback_group() is a function that gets the CallbackGroupStruct corresponding to a given callback_group_name.
It does not get all callback groups.

Reply:
I seem to have misunderstood something.
Fixed in https://github.com/tier4/caret_analyze/pull/511/commits/ce7a66f001a7fd44eb8fb3d6f4c7cdf0c219d764.